src/wbi/commands/remote.py

```diff
@@ -21,38 +20,61 @@ def add_args(parser):
         default="hello",
         help="Template to use for job script (default: hello)",
     )
+    parser.add_argument(
+        "--node",
+        type=str,
+        default="compute",
+        help="Specify if you want to execute command in compute node or the head node.",
+    )
     parser.add_argument(
         "-v", "--verbose", action="store_true", help="Increase verbosity"
     )
 
     return parser
 
 
-def main(args):
-
-    hostname, username = args.hostname, args.username
+def setup_ssh_client(hostname, username):
     client = paramiko.SSHClient()
     client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
     client.connect(hostname, username=username)
+    return client
 
-    job_id, remote_temp_stdout_path = submit(
-        template_name=args.template, mins=1, client=client
-    )
-    logger.info(f"Submitted job with ID {job_id}")
-    logger.info(f"Remote stdout path: {remote_temp_stdout_path}")
 
+def monitor_job(client, remote_temp_stdout_path=None):
     while True:
-        stdout = parse_remote_stdout(client, remote_temp_stdout_path)
+        stdout = get_remote_stdout(client, remote_temp_stdout_path)
         if "Hello" in stdout:
```

Comment:
Since we don't care what's actually in `stdout`, let's use the sentinel value of `None` from the function to indicate that no `stdout` is available (yet). (Of course the function itself would have to be modified slightly).

Simply having some `stdout` of course doesn't mean that the job is complete in `sbatch` mode (it may be a long running job), but we'll handle that later.

Reply:
Updated `parse_remote_stdout` function to return `None` if the file is not found. 
Updated `monitor_job` function to break out of the `while` if there is whitespace available, instead of looking for a definitive output like `Hello`.  The update looks like: 
```
elif " " in stdout:
     break
```